src/DNA/native/src/JIT_Execute.c

```diff
@@ -694,31 +703,31 @@ U32 JIT_Execute(tThread *pThread, U32 numInst) {
 JIT_CONV_I64_U64_start:
 JIT_CONV_U64_I64_start:
 	OPCODE_USE(JIT_NOP);
-JIT_NOP_end:
-JIT_CONV_R32_R32_end:
-JIT_CONV_R64_R64_end:
-JIT_CONV_I64_U64_end:
-JIT_CONV_U64_I64_end:
-JIT_GoNext_start:
+//JIT_NOP_end:
```

Comment:
@eloraiby Do you need to comment the ends, I was trying to keep the threaded code intact in case it's faster than the switch on some platforms? Is there another way to ignore the unused label warnings?

Reply:
Unless of course you just want to get rid of it.

Reply:
@ncave I can do it via a macro :), probably that's better for now. I would hold before cleaning up the code entirely, at least until we have a working x64 version. I haven't finished merging the upstream yet.